commit 86306dad1032b81dd653259434839eee92bda333
Author: Trond Myklebust <trond.myklebust@primarydata.com>
Date:   Fri Feb 17 18:42:32 2017 -0500

    NFSv4: Fix reboot recovery in copy offload
    
    commit 9d8cacbf5636657d2cd0dda17438a56d806d3224 upstream.
    
    Copy offload code needs to be hooked into the code for handling
    NFS4ERR_BAD_STATEID by ensuring that we set the "stateid" field
    in struct nfs4_exception.
    
    Reported-by: Olga Kornievskaia <aglo@umich.edu>
    Fixes: 2e72448b07dc3 ("NFS: Add COPY nfs operation")
    Signed-off-by: Trond Myklebust <trond.myklebust@primarydata.com>
    Signed-off-by: Anna Schumaker <Anna.Schumaker@Netapp.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1334_86306dad1032b81dd653259434839eee92bda333.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1334_86306dad1032b81dd653259434839eee92bda333.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
